### PR TITLE
Paul/more testing

### PR DESCRIPTION
**Pull request by paullinator**
### CHANGELOG

 none

### Dependencies

 none

### Requirements

If you have made **any** visual changes to the GUI. Make sure you have:

- [x] Tested on iOS device
- [ ] Tested on Android device
- [ ] Tested on small-screen device (iPod Touch)
- [ ] Tested on large-screen device (tablet)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205058055652274

### CODE REVIEW

File: src/components/tiles/Tile.tsx [hunk 1]
```diff
@@ -65,7 +65,7 @@ export class TileComponent extends React.PureComponent<Props> {
       )
     }
     return (
-      <TouchableWithoutFeedback onPress={this.handlePress} disabled={type === 'static'}>
+      <TouchableWithoutFeedback accessible={false} onPress={this.handlePress} disabled={type === 'static'}>
```
Comment (samholmes) [inline src/components/tiles/Tile.tsx:68]: What children were made accessible here?

File: src/components/themed/ThemedButtons.tsx [hunk 2]
```diff
@@ -92,7 +92,7 @@ export function RightChevronButton(props: { text: string; onPress: () => void; p
   const padding = sidesToPadding(mapSides(fixSides(paddingRem, 0), theme.rem))
 
   return (
-    <TouchableOpacity onPress={onPress}>
+    <TouchableOpacity accessible={false} onPress={onPress}>
```
Comment (samholmes) [inline src/components/themed/ThemedButtons.tsx:95]: What children were made accessible here?

File: src/components/themed/ShareButtons.tsx [hunk 3]
```diff
@@ -34,7 +34,7 @@ function ShareButton(props: { text: string; onPress: () => Promise<void>; icon:
   const [pending, handlePress] = usePendingPress(onPress)
 
   return (
-    <TouchableOpacity style={styles.button} onPress={handlePress} disabled={pending}>
+    <TouchableOpacity accessible={false} style={styles.button} onPress={handlePress} disabled={pending}>
```
Comment (samholmes) [inline src/components/themed/ShareButtons.tsx:37]: What children were made accessible here?

File: src/components/themed/ExchangedFlipInput2.tsx [hunk 4]
```diff
@@ -204,7 +204,7 @@ const ExchangedFlipInput2Component = React.forwardRef<ExchangedFlipInputRef, Pro
 
   return coreWallet != null ? (
     <>
-      <TouchableOpacity onPress={headerCallback} style={styles.headerContainer}>
+      <TouchableOpacity accessible={false} onPress={headerCallback} style={styles.headerContainer}>
```
Comment (samholmes) [inline src/components/themed/ExchangedFlipInput2.tsx:207]: What children were made accessible here?

File: src/components/scenes/RequestScene.tsx [hunk 5]
```diff
@@ -364,7 +364,7 @@ export class RequestSceneComponent extends React.Component<Props, State> {
               />
             )}
           />
-          <TouchableOpacity disabled={addressExplorerDisabled} onPress={this.handleAddressBlockExplorer}>
+          <TouchableOpacity accessible={false} disabled={addressExplorerDisabled} onPress={this.handleAddressBlockExplorer}>
```
Comment (samholmes) [inline src/components/scenes/RequestScene.tsx:367]: What children were made accessible here?

**Review by samholmes — CHANGES_REQUESTED** [review]
I have questions.

File: src/components/themed/ThemedButtons.tsx (hunk 2 above)
Comment (paullinator) [reply src/components/themed/ThemedButtons.tsx:95]: The EdgeText is made accessible

File: src/components/tiles/Tile.tsx (hunk 1 above)
Comment (paullinator) [reply src/components/tiles/Tile.tsx:68]: All the content within the Tile such as the title

File: src/components/themed/ShareButtons.tsx (hunk 3 above)
Comment (paullinator) [reply src/components/themed/ShareButtons.tsx:37]: The EdgeText is made accessible

File: src/components/scenes/RequestScene.tsx (hunk 5 above)
Comment (paullinator) [reply src/components/scenes/RequestScene.tsx:367]: The EdgeText is made accessible

File: src/components/themed/ExchangedFlipInput2.tsx (hunk 4 above)
Comment (paullinator) [reply src/components/themed/ExchangedFlipInput2.tsx:207]: The EdgeText is made accessible

**Review by samholmes — APPROVED** [review]

**Pull request merged**